src/Media.Plugin.WindowsPhone81/CameraCaptureUI.xaml.cs

```diff
@@ -196,7 +200,7 @@ private void CameraCaptureUI_Unloaded(object sender, RoutedEventArgs e)
 
 
 
-        StoreCameraMediaOptions options;
+        StoreCameraMediaOptions Options;
```

Comment:
Can you change this back to lower case and then just use a "this.options = options" below.